content/docs/extensions/pg-extensions.md

```diff
@@ -9,65 +9,66 @@ Neon supports the PostgreSQL extensions shown in the following table. For extens
 
 <a id="default-extensions/"></a>
 
-| Extension                | Extension Version (PostgreSQL 14)                               | Extension Version (PostgreSQL 15)                                |Note                                                                                                                |
```

Comment:
@danieltprice , I added here only one line, but it seems that my editor decided to format the whole file :( Feel free to update it